Compat error with array depencency

When compiling `bio`, I get this error:
```
    Checking bio v1.4.0
error[E0277]: `usize` is not an iterator
   --> /home/jakob/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bio-1.4.0/src/pattern_matching/pssm/mod.rs:299:20
    |
299 |         for row in scores.rows() {
    |                    ^^^^^^^^^^^^^ `usize` is not an iterator
    |
    = help: the trait `Iterator` is not implemented for `usize`
    = note: if you want to iterate between `start` until a value `end`, use the exclusive range syntax `start..end` or the inclusive range syntax `start..=end`
    = note: required for `usize` to implement `IntoIterator`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `bio` (lib) due to previous error
```
Which I believe stems from `scores.rows()` now returning `usize` not a view of rows.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,7 +30,7 @@ itertools = ">=0.8, <0.12"
 itertools-num = "0.1"
 bv = { version = "0.11", features = ["serde"] }
 bit-set = "0.5"
-ndarray= ">=0.13, <0.16"
+ndarray= ">=0.15, <0.16"
 lazy_static = "1.4"
 serde = "1.0"
 serde_derive = "1.0"

